lib/beacon/template/heex_transformer.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Beacon.Template.HEEx.HeexTransformer do
+  def transform(ast) do
```

Comment:
I think it should format the resulting heex code. Users will edit the template in the admin code editor and page builder at the same time, while the code in the code editor will most likely be formatted (or close to it), editing a page in the page builder will result in unformatted code causing a bad UX.

It's not a blocking issue for this PR but I think would be useful to call https://github.com/phoenixframework/phoenix_live_view/blob/fb111738d56745f37338867b9faea86eb9baa6e1/lib/phoenix_live_view/html_formatter.ex#L233